src/ghdcbot/bot.py

```
@@ -357,234 +353,171 @@ async def verify_link_cmd(interaction: discord.Interaction, github_username: str
     @tree.command(
-        name="verify",
-        description="Show your GitHub link verification status (read-only)",
+        name="profile",
+        description="Show contributor profile (GitHub, socials, roles; optional Discord member)",
         guild=discord.Object(id=guild_id),
     )
-    async def verify_cmd(interaction: discord.Interaction) -> None:
+    @app_commands.describe(
+        contributor="Discord member to view (omit for yourself)",
+    )
+    async def profile_cmd(
```

Ruff (0.15.21)

[warning] Too many branches (23 > 12)

(PLR0912)